libjob_queue/src/lsf_driver.c

```diff
@@ -839,7 +851,9 @@ static void lsf_driver_set_remote_server( lsf_driver_type * driver , const char
   }
 }
 
-
+void lsf_driver_add_exclude_hosts( lsf_driver_type * driver , char * excluded) {
+  stringlist_append_copy(driver->exclude_hosts, excluded);
```

Comment:
I guess it would make sense to allow the `excluded` argument to be a list:
```C
stringlist_type * host_list = stringlist_alloc_from_split( excluded, ", ");
for (int i=0; i < stringlist_get_size( host_list ); i++) {
      const char * excluded = stringlist_iget( host_list , i );
      ....
}